### PR TITLE
Added getbooking request info to allow user to get all the info on th…

### PR DESCRIPTION
**Pull request by fangpinsern**
…eir request

so people can see what happen to their request

### CODE REVIEW

File: src/routes/bookingRequest.routes.js [hunk 1]
```diff
@@ -19,10 +23,12 @@ router.get("/ping", (req, res, next) => {
   return res.send("Successfully inside booking request routes");
 });
 
-router.post(
+router.get("/get", validationHelper());
```
Comment (chwennnn) [inline src/routes/bookingRequest.routes.js:26]: what's this?

**Review by chwennnn — APPROVED** [review]
minor comments

File: src/routes/bookingRequest.routes.js (hunk 1 above)
Comment (fangpinsern) [reply src/routes/bookingRequest.routes.js]: shouldnt be there. forgot to delete

Comment (fangpinsern) [reply src/routes/bookingRequest.routes.js]: removed

**Pull request merged**